### ISSUE TITLE
[Suggestion]: no suffocation in beams

### ISSUE THREAD

**Issue by MrRoomba69** [nonamecrackers2/crackers-wither-storm-mod#1568]
### Description of feature request

i would like to see a config where you can turn on and off mob suffocation in tractor beams of the storm.

**Comment by GrandmaPork** [comment]
Suffocation?

**Comment by TaintedBlockmObile** [comment]
Suffocating in what? The beam isn't a solid.

**Comment by nonamecrackers2** [comment]
Yeah don't know what you mean by "suffocation." Could you elaborate?

**Comment by MrRoomba69** [comment]
loss of o2 with aquatic mobs

**Comment by TaintedBlockmObile** [comment]
> loss of o2 with aquatic mobs

I uh.. don't think the tractor beams removed oxygen tho? It's not like a solid wall against oxygen

**Comment by myron-obsolete-account** [comment]
> loss of o2 with aquatic mobs

fish and similar mobs always suffocate when out of water, it has nothing to do with the Wither Storm, basically this is irrelevant.

**Comment by myron-obsolete-account** [comment]
I believe he wants a config that water mobs can survive out of water when being sucked up which doesn't really make it up for me in my brain

**Comment by BluePineappleGuy** [comment]
> I believe he wants a config that water mobs can survive out of water when being sucked up which doesn't really make it up for me in my brain

he wants them not to suffocate while in the beams so the storm sucks them up instead of them dying mid-way.

**Comment by TaintedBlockmObile** [comment]
> > I believe he wants a config that water mobs can survive out of water when being sucked up which doesn't really make it up for me in my brain
> 
> he wants them not to suffocate while in the beams so the storm sucks them up instead of them dying mid-way.

They don't suffocate tho..??

**Comment by myron-obsolete-account** [comment]
> > > I believe he wants a config that water mobs can survive out of water when being sucked up which doesn't really make it up for me in my brain
> > 
> > 
> > he wants them not to suffocate while in the beams so the storm sucks them up instead of them dying mid-way.
> 
> They don't suffocate tho..??

they don't?

**Comment by BluePineappleGuy** [comment]
> > > I believe he wants a config that water mobs can survive out of water when being sucked up which doesn't really make it up for me in my brain
> > 
> > 
> > he wants them not to suffocate while in the beams so the storm sucks them up instead of them dying mid-way.
> 
> They don't suffocate tho..??

pretty sure they do die in the beams

**Comment by TaintedBlockmObile** [comment]
> > > > I believe he wants a config that water mobs can survive out of water when being sucked up which doesn't really make it up for me in my brain
> > > 
> > > 
> > > he wants them not to suffocate while in the beams so the storm sucks them up instead of them dying mid-way.
> > 
> > 
> > They don't suffocate tho..??
> 
> pretty sure they do die in the beams

If they die in the beams it's either because of either being hit by a wither skull and get the wither effect, or hit by another entity. They don't suffocate in the beam.

**Comment by myron-obsolete-account** [comment]
> > > > > I believe he wants a config that water mobs can survive out of water when being sucked up which doesn't really make it up for me in my brain
> > > > 
> > > > 
> > > > he wants them not to suffocate while in the beams so the storm sucks them up instead of them dying mid-way.
> > > 
> > > 
> > > They don't suffocate tho..??
> > 
> > 
> > pretty sure they do die in the beams
> 
> If they die in the beams it's either because of either being hit by a wither skull and get the wither effect, or hit by another entity. They don't suffocate in the beam.

they suffocate because they run out of air. the OP said aquatic mobs, fish and dolphins for example are aquatic mobs, which die when exposed to air to a long time. this is normal behaviour of those mobs

**Comment by MrRoomba69** [comment]
> > I believe he wants a config that water mobs can survive out of water when being sucked up which doesn't really make it up for me in my brain
> 
> he wants them not to suffocate while in the beams so the storm sucks them up instead of them dying mid-way.

yes, that's what I'm requesting

**Comment by GrandmaPork** [comment]
Do the fish items get eaten on death? This may reduce lag by clearing up items

**Comment by MrRoomba69** [comment]
i would like to see some progress on if this gets added or not
